src/compass/utils/validate_cslc.py

```diff
@@ -110,12 +110,32 @@ def compare_cslc_products(file_ref, file_sec):
         return
 
     # Compare amplitude of reference and generated CSLC products
```

Comment:
```suggestion
    # Compare amplitude of reference and generated CSLC products
    np.seterr(invalid='ignore')
```
In case you want to suppress the warning message like below:
> RuntimeWarning: invalid value encountered in divide